docs/admin/deployments.rst

```diff
@@ -26,7 +26,7 @@ The following examples assume you have a working Docker environment, with
 
    .. code-block:: sh
 
-        git clone https://github.com/WeblateOrg/docker-compose.git weblate-docker
+        git clone --recurse-submodules https://github.com/WeblateOrg/docker-compose.git weblate-docker
```

Comment:
https://github.com/WeblateOrg/docker-compose is the docker-compose module, you probably mean https://github.com/WeblateOrg/docker, which is on the other side (intentionally) not used in the install instructions.

Reply:
Oh yes, totally !
I followed an old link and didn't seen that we now have to clone docker-compose.
Sorry, thank you !